onnxruntime/python/session.py

```diff
@@ -24,23 +24,22 @@ def __init__(self, path_or_bytes, sess_options=None):
         self._enable_fallback = True
 
     def _load_model(self, providers=[]):
-        if self._sess_options:
+        if isinstance(self._path_or_bytes, str): 
             self._sess = C.InferenceSession(
-                self._sess_options, C.get_session_initializer())
-        else:
-            self._sess = C.InferenceSession(
-                C.get_session_initializer(), C.get_session_initializer())
-
-        if isinstance(self._path_or_bytes, str):
-            self._sess.load_model(self._path_or_bytes, providers)
+                self._sess_options if self._sess_options else C.get_default_session_options(), 
+                self._path_or_bytes, True)
         elif isinstance(self._path_or_bytes, bytes):
-            self._sess.read_bytes(self._path_or_bytes, providers)
-        elif isinstance(self._path_or_bytes, tuple):
+            self._sess = C.InferenceSession(
+                self._sess_options if self._sess_options else C.get_default_session_options(), 
+                self._path_or_bytes, False)
+        # elif isinstance(self._path_or_bytes, tuple):
```

Comment:
Can't seem to find `load_model_no_init()` anywhere. It seems like even in our current release package, we would fail if the InferenceSession ctor() is invoked with a tuple. Commenting it out for now.